src/search.ts

```diff
@@ -0,0 +1,69 @@
+import * as glob from '@actions/glob'
+import {debug} from '@actions/core'
+import {lstatSync} from 'fs'
+import {dirname} from 'path'
+
+export interface SearchResult {
+  filesToUpload: string[]
+  rootDirectory: string
+}
+
+function getDefaultGlobOptions(): glob.GlobOptions {
+  return {
+    followSymbolicLinks: true,
```

Comment:
Is there any protection around symbolic link loops if we follow link?

Reply:
Yes there are protections: https://github.com/actions/toolkit/blob/d17d4a916377cc569a5c642b9d2f56c23d1ab620/packages/glob/src/internal-globber.ts#L231